Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/PressableButtonInspector.cs

```diff
@@ -1,6 +1,7 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using Microsoft.MixedReality.Toolkit.Input;
```

Comment:
When I make the planes editable, i get the following error:

ArgumentException: Getting control 18's position in a group with only 18 controls when doing repaint
Aborting
UnityEngine.GUILayoutGroup.GetNext () (at C:/buildslave/unity/build/Modules/IMGUI/LayoutGroup.cs:117)
UnityEngine.GUILayoutUtility.DoGetRect (System.Single minWidth, System.Single maxWidth, System.Single minHeight, System.Single maxHeight, UnityEngine.GUIStyle style, UnityEngine.GUILayoutOption[] options) (at C:/buildslave/unity/build/Modules/IMGUI/GUILayoutUtility.cs:461)

![image](https://user-images.githubusercontent.com/168492/58219423-afb54100-7cbf-11e9-8645-886d9ff19054.png)

Reply:
that's caused by hiding/showing ui elements in the OnInspectorGUI method - I've replaced all those checks with disabled groups , so we won't hide the options anymore but instead show them as grayed out (disabled)